WcaOnRails/app/models/competition.rb

```diff
@@ -27,6 +27,7 @@ class Competition < ApplicationRecord
   has_many :wcif_extensions, as: :extendable, dependent: :delete_all
   has_many :bookmarked_competitions, dependent: :delete_all
   has_many :bookmarked_users, through: :bookmarked_competitions, source: :user
+  belongs_to :main_event, class_name: "Event"
```

Comment:
It's a bit weird to have this be a `belongs_to` when really it's the competition which has one main event (but yeah given the key is here it's the correct thing).
As far as I understand the code, calling `competition.main_event` would fire a select right?
Couldn't we just implement the `main_event` attribute like that:
```ruby
def main_event
  Event.c_find(main_event_id)
end
```